src/client/cardable.test.ts

```diff
@@ -1,23 +1,20 @@
-import anyTest, {TestInterface} from 'ava'
+import test from 'ava'
 import Axios, {AxiosRequestConfig, AxiosResponse} from 'axios'
-import Cardable from './cardable'
+import Client from '../client'
+import WithCardable from './cardable'
 import {SeqMode} from '../client.enum'
 import {IDeleteCardResult, ISaveCardResult, ISearchCardResult} from './cardable.interface'
 
-interface Context {
-  card: Cardable
-}
+const Cardable = WithCardable(Client)
+let card: any
```

Comment:
I had to change these, because 
```ts
const Cardable = WithCardable(Client)
interface Context {
  card: Cardable
}
```
is invalid in TS, Cardable is a value, not a type. Is there another way?

Reply:
My idea is using type-inference.

```ts
let card = new Cardable()
```